### DISCUSSION TITLE
SDXL Support Q&A

### DISCUSSION THREAD

**Discussion by Aptronymist** [vladmandic/sdnext#1636] [Q&A] [answered]
Starting this up, as everyone is dying to get this going properly! 
Feel free to contribute Q&A pairs if you have run into issues that you've resolved or assisted others with and you would like to contribute them, we will very much appreciate it, as will the people you help with your experience.

**Comment by Aptronymist** [comment]
**Q: Is img2img supported with SDXL?**

A: Basic img2img functions are currently unavailable as of today, due to architectural differences, however it is being worked on. The more advanced functions, inpainting, sketching, those things will take a bit more time. 

Keep an eye on the changelog and announcements!

**Comment by Aptronymist** [comment]
**Q: When I'm generating images with SDXL, it freezes up near the end of generating and sometimes takes a few minutes to finish. Is there something wrong?**

A: No, with SDXL, the freeze at the end is actually rendering from latents to pixels using built-in VAE.
Diffusers currently does not report the progress of that, so the progress bar has nothing to show. 
That last step does have VRAM spike so it can cause a spillover from VRAM to shared memory and then instead of executing in 1-2 seconds, it can take a looong time.

**Comment by Tillerz** [comment]
Q: my images look really weird and low quality, compared to what I see on the internet.

A: SDXL has been trained with 1024x1024 images (hence the name XL), you probably try to render 512x512 with it, stay with (at least) 1024x1024 base image size.

**Comment by fcabanski** [comment]
Is there a guide to installing the model to run with vladomatic.

Downloaded it into StableDiffusion Models dir. Selected it in vlad. Error: model not loaded.

**Reply by vladmandic** [reply]
A) wiki
B) discord for questions

**Reply by fcabanski** [reply]
Thanks

https://stable-diffusion-art.com/sdxl-model/

**Reply by cgidesign-de** [reply]
maybe here is a solution:
https://github.com/vladmandic/automatic/discussions/1691

**Reply by Aptronymist** [reply]
We have our own [installation guide](https://github.com/vladmandic/automatic/wiki/Diffusers), which I updated the other day.

**Comment by DirtyHamster** [comment]
Is there a good guide or working notes/paper on what is actually trained inside of each of the models they are using? I know they specified base model and the other one for finer details but language wise that is still obscure in meaning in realistic terms and examples.

**Reply by Aptronymist** [reply, marked as answer]
I feel like the place to find that information would be on the StabilityAI website.

**Comment by EriIaz** [comment]
I have the pruned weights of the SDXL 0.9, and they work well in Comfy. Is it possible to use them in SD.Next instead of 12GB models from Huggingface? Those checkpoints are bigger than my VRAM ;D

**Reply by vladmandic** [reply]
yes, they work fine
